src/settings_gui.cpp

```diff
@@ -214,6 +241,14 @@ struct GameOptionsWindow : Window {
 				}
 				break;
 
+			case WID_GO_REFRESH_RATE_DROPDOWN: // Setup refresh rate dropdown
+				// TODO: Check for actual available
+				for (uint i = 0; i < refresh_rates.size(); i++) {
+					std::string string = std::to_string(refresh_rates[i]) + " Hz";
```

Comment:
You trigger my OCD .. `" Hz"` vs `"Hz"` on line 164 :D (space vs no-space)

Reply:
Fixed in 6b9c050 with a default value.